code/modules/jobs/job_types/medical_doctor.dm

```diff
@@ -11,7 +11,7 @@
 
 	outfit = /datum/outfit/job/doctor
 
-	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_PHARMACY, ACCESS_CHEMISTRY, ACCESS_GENETICS, ACCESS_VIROLOGY, ACCESS_MECH_MEDICAL, ACCESS_MINERAL_STOREROOM)
```

Comment:
This is skeleton acces, they only get chem acces when pop is like below 20

Reply:
I'm an idiot